assets/rules/socialMediaBlockRules.js

```diff
@@ -0,0 +1,192 @@
+      }
+    },
+    {
+      "id": 582,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*instagram*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 583,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*snapchat*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 584,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*tiktok*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 585,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*linkedin*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 586,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*reddit*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 587,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*pinterest*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 588,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*discord*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 589,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*tumblr*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 590,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*telegram*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 591,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*whatsapp*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 592,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*weibo*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 593,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*vk*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 594,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*line*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 595,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*skype*/",
+        "resourceTypes": ["main_frame"]
+      }
+    },
+    {
+      "id": 596,
+      "priority": 1,
+      "action": { "type": "block" },
+      "condition": {
+        "urlFilter": "/*youtube*/",
```

Comment:
we can allow youtube. its not a social media site. 
If we can block youtube and only allow youtube kids, that would be a good to have feature

Reply:
YouTube can't be blocked

Reply:
so lets remove the rule from here. we should be able to block it though